aspnetcore/web-api/advanced/conventions/sample/ApiConventions.csproj

```diff
@@ -1,11 +1,8 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+<Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
+    <IncludeOpenAPIAnalyzers>true</IncludeOpenAPIAnalyzers>
```

Comment:
This is the fix for the analyzer's NRE and is documented [here](https://docs.microsoft.com/en-us/aspnet/core/web-api/advanced/analyzers?view=aspnetcore-6.0&tabs=visual-studio#reference-the-analyzer-package).